pages/dao/[symbol]/proposal/components/instructions/ProgramUpgrade.tsx

```diff
@@ -102,7 +102,14 @@ const ProgramUpgrade = ({
       index
     )
   }, [form])
-
+  useEffect(() => {
+    if (governedProgramAccounts.length === 1) {
+      handleSetForm({
+        value: governedProgramAccounts[0],
+        propertyName: 'governedAccount',
+      })
+    }
+  }, [governedProgramAccounts.length])
```

Comment:
so this is just to auto fill in that field if it knows there is only one right, so there is no need for fallback the user can just select one otherwise?

Reply:
yes

Reply:
oh i will change that i have idea for better solution